.gitignore

```diff
@@ -7,6 +7,12 @@ tests/soaktest-*
 *.gz
 /.vscode
 /shotover-proxy/example-configs/redis-tls/certs
+/shotover-proxy/example-configs/cassandra-tls/certs/*.jks
+/shotover-proxy/example-configs/cassandra-tls/certs/*.p12
+/shotover-proxy/example-configs/cassandra-tls/certs/*.key
+/shotover-proxy/example-configs/cassandra-tls/certs/*.srl
+/shotover-proxy/example-configs/cassandra-tls/certs/*.csr
+/shotover-proxy/example-configs/cassandra-tls/certs/*.crt
```

Comment:
Any reason to use specific file extensions instead of excluding the whole folder like for Redis?

Doesn't matter, just curious.

Reply:
Nvm, I missed the fact that the script is in the same folder.